test/jdk/javax/swing/JPopupMenu/8075063/ContextMenuScrollTest.java

```diff
@@ -93,6 +87,11 @@ public static void main(String[] args) throws Exception {
 
 
     public static void createGUI() {
+        JMenuItem paste;
+        JMenuItem delete;
+        JMenuItem selectAll;
+        JMenuBar menuBar;
```

Comment:
Why at the top of the method instead of the first usage?
What about `undo`, `redo`, `cut`, `copy`? I cannot see they're used anywhere else but this method too.